api/Services/Mapper.cs

```diff
@@ -8,14 +8,21 @@ public class Mapper : Profile
     {
         public Mapper()
         {
-            CreateMap<HotelCard, HotelEntity>();
-            CreateMap<HotelEntity, HotelCard>();
+            CreateMap<HotelEntity, HotelCard>()
+                .ReverseMap();
 
-            CreateMap<AddHotelRepresentation, HotelEntity>();
-            CreateMap<HotelEntity, AddHotelRepresentation>();
+            CreateMap<HotelEntity, Hotel>()
+                .ReverseMap();
 
-            CreateMap<HotelRepresentation, HotelEntity>();
-            CreateMap<HotelEntity, HotelRepresentation>();
+            CreateMap<HotelEntity, AddHotel>()
+                .ReverseMap();
+
+            CreateMap<HotelEntity, EditHotel>()
+                .ReverseMap()
+                .ForAllMembers(opts => opts.Condition((src, dest, srcMember) => srcMember != null));
```

Comment:
Why?

Reply:
I map editHotel to hotelEntity as source and destination. source model does not have all fields of destination and I want such fields to stay as they were in destination